lib/cadet_web/controllers/grading_controller.ex

```diff
@@ -74,12 +78,21 @@ defmodule CadetWeb.GradingController do
   swagger_path :index do
     get("/grading")
 
-    summary("Get a list of all submissions with current user as the grader. ")
+    summary("Get a list of all submissions")
```

Comment:
Not sure why this line needs to be changed.

Reply:
Yup reverted (wanted to say that by default it gives all submissions but they can infer that from the params)